fiona/_crs.pyx

```diff
@@ -53,12 +58,11 @@ def crs_to_wkt(crs):
             proj_b = proj.encode('utf-8')
             proj_c = proj_b
             OSRImportFromProj4(cogr_srs, proj_c)
+
     else:
         raise ValueError("Invalid CRS")
 
-    # Fixup, export to WKT, and set the GDAL dataset's projection.
-    OSRFixup(cogr_srs)
```

Comment:
I think this OSRFixup call should be still used in GDAL < 3.0. Supporting both version scenarios is probably complicated, however.

Reply:
Do you see any harm in having  osr_set_traditional_axis_mapping_strategy call OSRFixup for GDAL versions 1 and 2 @hobu ? That's one way to bring it back in.

Reply:
No harm, but that's not quite where you want it. You want it right after any ImportFromWkt-type calls happen where the input might be ESRI content.